### ISSUE TITLE
[fsmtest] DISPLAY is not correctly passed on to some components

### ISSUE THREAD

**Issue by marc-hanheide** [strands-project/strands_ci#6]
I see this in the log:

```
11:44:48 DEBUG [ProgramExecutor.py@120]: collect-logs [?]: Making all the observer threads start.
11:44:48 DEBUG [ProcessObserver.py@51]: collect-logs:27478@pid: Starting (blocking True)!
11:44:48 DEBUG [PTYbufferCleaner.py@38]: Writing: "Error: Can't open display: :0::100
11:44:48 DEBUG [PTYbufferCleaner.py@38]: Writing: "
11:44:48 ERROR [PTYbufferCleaner.py@45]: ERROR: Process became un-pollable!
11:44:50 ERROR [ProcessObserver.py@57]: collect-logs:27478@pid: Observer failed! No internal OK found (PID not found). Process collect-logs is dead (ongoing-mode).
```

The respective config looks like this:

```
[component-5]
name = collect-logs
command = xeyes
path = /usr/bin/
execution_host = localhost
check_execution = True
check_type = pid
timeout = 2
blocking = True
ongoing = True
criteria = 
```

and my `[environment]` contains `DISPLAY=:100`

**Comment by warp1337** [comment]
This is probably a duplicate of #4. As I said in the other issue, export it in the shell before starting fsmt. Another option would be the manually fixing the scxml file, until I have fixed the iniparser

**Comment by warp1337** [comment]
Please consider the new release 0.12 and again :)
